docs/fastdds/statistics/dds_layer/domainParticipant.rst

```diff
@@ -123,3 +148,7 @@ The following examples show how to use all the previous methods:
 
 Be aware that automatically enabling the statistics DataWriters using all these methods implies using the recommended
 QoS profile |STATISTICS_DATAWRITER_QOS-api|. For more information, please refer to :ref:`statistics_datawriter_qos`.
+However, if an XML profile is defined, the QoS applied are those defined in the profile,
+and for those QoS that are not specified in that profile, the default library QoS are applied
+(see :ref:`dds_layer_publisher_dataWriterQos` for the standard eProsima's DataWriter QoS)
+, and not the recommended QoS for the Statistics DataWriters.
```

Comment:
In order to improve readability I suggest leaving the comma at the end of the previous line

Reply:
Done: b5f68b0c Refs #15014: Comma.